checks/raw/shell_download_validate.go

```diff
@@ -507,6 +524,22 @@ func isUnpinnedPipInstall(cmd []string) bool {
 			break
 		}
 
+		if strings.EqualFold(cmd[i], "--no-deps") {
```

Comment:
do you mind adding as comment some of the PR description, including the links to the relevant doc?

Reply:
Done. Can you take a look?

Reply:
LG, thanks